samples/drivers/led_pwm/src/main.c

```diff
@@ -74,33 +73,25 @@ static void run_led_test(const struct device *led_pwm, uint8_t led)
 		}
 		k_sleep(FADE_DELAY);
 	}
+	LOG_INF("  Increased brightness gradually");
```

Comment:
So now we get something useful / parsable for a script, but useless for a human :)

With this message printed after the test, it will be difficult for a human to be sure that the LED behaved correctly.